src/dmd/target.d

```diff
@@ -900,6 +900,17 @@ extern (C++) struct Target
             || global.params.isDragonFlyBSD
             || global.params.isSolaris;
     }
+
+    /**
+     * Returns: a name for the current architecture.
+     */
+    extern(D) string architectureName()
```

Comment:
It would be more useful if this was `extern(C++)`, or it returned a field that could be set by the glue/backend.

Reply:
Okay, I see. Changed it to a field.